src/TraceEvent/RegisteredTraceEventParser.cs

```diff
@@ -1076,6 +1078,88 @@ internal struct TRACE_EVENT_INFO
             public EVENT_PROPERTY_INFO EventPropertyInfoArray;  // Actually an array, this is the first element.  
         }
 
+#if NETSTANDARD2_0
```

Comment:
I have made the change I suggested above to remove the dependency to System.Diagnostics.Eventing.EventDescriptor  with PR #357